<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PJobWindow</class>
 <widget class="QMainWindow" name="PJobWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="labConnectInfo">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string>ConnectInfo</string>
          </property>
          <property name="textFormat">
           <enum>Qt::MarkdownText</enum>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="openExternalLinks">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>page1</cstring>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="labRuleName">
            <property name="text">
             <string>选择一个规则</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox"/>
          </item>
          <item>
           <widget class="QDialogButtonBox" name="butBox">
            <property name="standardButtons">
             <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>调度周期</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="keyboardTracking">
             <bool>true</bool>
            </property>
            <property name="showGroupSeparator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="suffix">
             <string> 秒/次</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="stepType">
             <enum>QAbstractSpinBox::DefaultStepType</enum>
            </property>
            <property name="value">
             <number>60</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkSuspend">
            <property name="toolTip">
             <string>取消选择正常调度此任务</string>
            </property>
            <property name="text">
             <string>暂停后台调度</string>
            </property>
            <property name="tristate">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>658</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QComboBox" name="combTimeRang">
            <item>
             <property name="text">
              <string>1天</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2天</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4天</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7天</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>数据范围</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>下次运行时间</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDateTimeEdit" name="dateTimeEdit">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="specialValueText">
             <string/>
            </property>
            <property name="accelerated">
             <bool>false</bool>
            </property>
            <property name="keyboardTracking">
             <bool>true</bool>
            </property>
            <property name="showGroupSeparator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="currentSection">
             <enum>QDateTimeEdit::YearSection</enum>
            </property>
            <property name="displayFormat">
             <string>yyyy/MM/dd HH:mm:ss</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCommandLinkButton" name="btnRunNow">
            <property name="toolTip">
             <string>立即启动当前任务，并不会影响后台调度时间</string>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>立即运行任务</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tabBatch">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>采样间隔</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="combFlagCol">
            <property name="editable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="checkBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>  监控变化率</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="btnSlide">
            <property name="text">
             <string>实时监控</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spinBox_2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="keyboardTracking">
             <bool>true</bool>
            </property>
            <property name="showGroupSeparator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="suffix">
             <string> 秒/次</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="stepType">
             <enum>QAbstractSpinBox::DefaultStepType</enum>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>数据列</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>主键列</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="combDataCol"/>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QCommandLinkButton" name="btnRunNow_3">
            <property name="toolTip">
             <string>立即启动当前任务，并不会影响后台调度时间</string>
            </property>
            <property name="text">
             <string>绘制图像</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tabcomment">
        <attribute name="title">
         <string>任务备注</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QPlainTextEdit" name="editComment">
           <property name="plainText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabWidgetPage2">
        <attribute name="title">
         <string>明细结果</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QTableView" name="tabResult">
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="gridStyle">
            <enum>Qt::SolidLine</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabWidgetPage3">
        <attribute name="title">
         <string>图表视图</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="PChartView" name="chartView">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PChartView</class>
   <extends>QGraphicsView</extends>
   <header>PulseUI/pchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
